java/merchant-client/src/main/java/no/bankaxept/epayment/client/merchant/MerchantClient.java

```diff
@@ -41,8 +32,23 @@ public MerchantClient(BaseClient baseClient) {
     this.baseClient = baseClient;
   }
 
-  public MerchantClient(String baseurl, String apimKey, String username, String password) {
-    this.baseClient = new BaseClient.Builder(baseurl).apimKey(apimKey).withScheduledToken(username, password).build();
+  public MerchantClient(
+      URL authorizationServerUrl,
```

Comment:
Is taking in a URL instead of a string supposed to prevent malformed URLs being passed in?

Reply:
Yes